src/v/cluster_link/model/types.cc

```diff
@@ -286,7 +286,18 @@ fmt::iterator shadow_topic_report_request::format_to(fmt::iterator it) const {
 
 fmt::iterator
 shadow_topic_partition_leader_report::format_to(fmt::iterator it) const {
-    return fmt::format_to(it, "{{ partition: {} }}", partition);
+    return fmt::format_to(
+      it,
+      "{{ partition: {}, source_partition_start_offset: {}, "
+      "source_partition_high_watermark: {}, "
+      "source_partition_last_stable_offset: {}, last_update_time: {}, "
+      "shadow_partition_high_watermark: {} }}",
+      partition,
+      source_partition_start_offset,
+      source_partition_high_watermark,
+      source_partition_last_stable_offset,
+      last_update_time,
```

Comment:
I think now() - time in ms is probably more human readable.

Reply:
could render epoch time?